src/main/java/io/sudhakar/student/controller/RestTemplateController.java

```diff
@@ -0,0 +1,76 @@
+package io.sudhakar.student.controller;
+
+import io.sudhakar.student.dto.Employee;
+import io.sudhakar.student.dto.ServiceResponse;
+import io.sudhakar.student.dto.StudentResponse;
+import io.sudhakar.student.service.impl.RestTemplateServiceImpl;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@Slf4j
+@RestController
+@RequestMapping("/employee")
+public class RestTemplateController {
+    private final RestTemplateServiceImpl restTemplateService;
+
+    public RestTemplateController(RestTemplateServiceImpl restTemplateService) {
+        this.restTemplateService = restTemplateService;
+    }
+
+    @GetMapping
+    public ResponseEntity<StudentResponse<?>> getAllEmployee(@RequestParam("urlName") String urlName) {
```

Comment:
don't take this as requestParam 
hardcode it in service

Reply:
hard coded in RestTemplateServiceImpl class